app/views/layouts/application.html.erb

```diff
@@ -4,13 +4,15 @@
     <title>ReviewSite8</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag    'application', media: 'all' %>
+    <%= javascript_include_tag 'application' %>
   </head>
 
   <body>
+    <% flash.each do |key, value| %>
+      <%= value %>
+    <% end %>
```

Comment:
You may want to consider wrapping those `value`s in an HTML element. A `<p>`, or even a `<div>` would be good.